src/pages/Main/Main.page.jsx

```diff
@@ -0,0 +1,166 @@
+    // vertical padding + font size from searchIcon
+    paddingLeft: `calc(1em + ${theme.spacing(4)})`,
+    transition: theme.transitions.create('width'),
+    width: '100%',
+    [theme.breakpoints.up('md')]: {
+      width: '20ch',
+    },
+  },
+}));
+
+const MyAppBar = () => {
+  return (
+    <AppBar position="fixed">
+      <Toolbar>
+        <IconButton
+          size="large"
+          edge="start"
+          color="inherit"
+          aria-label="open drawer"
+          sx={{ mr: 2 }}
+        >
+          <MenuIcon />
+        </IconButton>
+        <Search>
+          <SearchIconWrapper>
+            <SearchIcon />
+          </SearchIconWrapper>
+          <StyledInputBase
+            placeholder="wizeline"
+            inputProps={{ 'aria-label': 'search' }}
+          />
+        </Search>
+        <Box sx={{ flexGrow: 1 }} />
+        <Box sx={{ display: { xs: 'none', md: 'flex' } }}>
+          <FormGroup sx={{ justifyContent: 'center' }}>
+            <FormControlLabel disabled control={<Switch />} label="Dark mode" />
+          </FormGroup>
+          <IconButton
+            size="large"
+            edge="end"
+            aria-label="account of current user"
+            aria-haspopup="true"
+            color="inherit"
+          >
+            <AccountCircle />
+          </IconButton>
+        </Box>
+      </Toolbar>
+    </AppBar>
+  );
+};
+
+const Content = () => {
+  const [data, setData] = useState([]);
+  const getData = () => {
+    fetch('youtube-videos-mock.json', {
+      headers: {
+        'Content-Type': 'application/json',
+        Accept: 'application/json',
+      },
+    })
+      .then((response) => {
+        return response.json();
+      })
+      .then((myJson) => {
+        setData(myJson);
+      });
+  };
+
+  useEffect(() => {
+    getData();
+  }, []);
+  return (
+    <Grid
+      container
+      direction="row"
+      justifyContent="center"
+      alignItems="center"
+      spacing={2}
+      sx={{ padding: '12px' }}
+    >
+      {data.items &&
+        data.items.length > 0 &&
+        data.items.map((item) => (
+          <Grid item key={item.etag}>
+            <Card sx={{ maxWidth: 345, maxHeight: 345, width: 345, height: 345 }}>
+              <CardMedia
+                component="img"
+                height="140"
+                image={item.snippet.thumbnails.default.url}
+              />
+              <CardContent>
+                <Typography gutterBottom variant="h5" component="div">
+                  {item.snippet.title}
+                </Typography>
+                <Typography variant="body2" color="text.secondary">
+                  {item.snippet.description}
+                </Typography>
+              </CardContent>
+            </Card>
+          </Grid>
+        ))}
+    </Grid>
+  );
+};
+
+function MainPage() {
+  return (
+    <Box sx={{ flexGrow: 1 }}>
+      <MyAppBar />
+      <Toolbar />
+      <Content />
+    </Box>
+  );
+}
```

Comment:
Al mover este contenido al `HomePage` este componente no va a ser necesario, por lo que puedes borrarlo una vez lo migres para allá.

No tengo problema con tener un toolbar extra aquí para resolver el problema de que tu content se va detrás de `MyAppBar`, pero si lo vas a dejar allí sugeriría que lo incluyas en el return del componente `MyAppBar` al final, para que cuando incluyas tu contenido en el home se lean solo los que si sabemos que se muestran. Algo así.
```suggestion
function MainPage() {
  return (
    <Box sx={{ flexGrow: 1 }}>
      <MyAppBar />
      <Content />
    </Box>
  );
}
```